docs/technical_reference/unit_models/zero_order_unit_models/smp_zo.rst

```diff
@@ -13,8 +13,8 @@ See documentation for :ref:`Helper Methods for Electricity Demand<electricity_me
 
 Costing Method
```

Comment:
Small thing, but I realized that "SMP" is not capitalized in the toc tree nor within the rst file. To change this, you can add an entry for smp in `title_exceptions` in the automation script. I remember some talk about removing this unit all together, but since we have it here, might as well give it a name. @kurbansitterley do you remember what "SMP" stood for in WT3?

Reply:
Hm no I don't remember... soluble microbial products is the only SMP acronym I know.

Reply:
WT3 doesn't seem to know either.. https://github.com/search?q=repo%3ANREL%2FWaterTAP3%20SMP&type=code

Reply:
I elected to just capitalize SMP [3de7d0f](https://github.com/watertap-org/watertap/pull/1081/commits/3de7d0f8875f40b2f21cd1e16d1caee6d30a4eb6)